### DISCUSSION TITLE
关于使用自己的 secret 进行 AES 加密方式的小白解决办法

### DISCUSSION THREAD

**Discussion by ChirmyRam** [spencerwooo/onedrive-vercel-index#234] [Show and tell]
可能很多像我一样的小白想使用自己的secret，但又不知道如何加密。我折腾了很久才摸索出来，使用这个在线加密工具 [https://www.dute.org/aes](https://www.dute.org/aes) ，原始文本填入自己的secret，秘钥填入`onedrive-vercel-index`，`加/解密模式`选择`CBC (Cipher Block Chaining, Default)`，点击`AES 加密`就可以了。我也是这样将作者内置的secret反向解密出来的，加密方式试了一遍又一遍~

**Comment by HaoTian22** [comment]
谢谢，我之前用了几个工具加密出来都不一样的😂
这个终于成功了

**Reply by MyCatRubyLoveMe** [reply]
谢谢，一次成功，太棒了

**Comment by spencerwooo** [comment]
FYI: https://onedrive-vercel-index.spencerwoo.com/docs/advanced#modify-configs-in-apiconfigjs

<img width="897" alt="CleanShot 2022-02-03 at 21 35 21@2x" src="https://user-images.githubusercontent.com/32114380/152353012-93694b7a-9381-45eb-b359-870b28054c6b.png">

**Reply by ChirmyRam** [reply]
nice

**Comment by SemiZhang** [comment]
把加密密钥公布出来真的好吗？

**Reply by ChirmyRam** [reply]
这是作者当初在README公开出来的。实际上连client id和secret完全公开都没什么影响。
